rlschool/A1_robot/envs/env_builder.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
```

Comment:
What's the difference between this file and `env_builder_old.py` ?
Remove the copyright notice here. For the copyright notice of the third-party code, please refer to this file: 
https://github.com/PaddlePaddle/PARL/blob/369da17e23812131d963c2e816d340ba2fc089fa/examples/ES/noise.py#L1